### ISSUE TITLE
Custom implementations of IBrowser being ignored by Auth0 Oidc for Android and iOS.

### ISSUE THREAD

**Issue by badavid85** [auth0/auth0-oidc-client-net#10]
On Android and iOS, Is there any specific reason for ignoring custom implementations of IBrowser on the Auth0Client class?

```
#if __IOS__
    RedirectUri = $"{Foundation.NSBundle.MainBundle.BundleIdentifier}://{options.Domain}/ios/{Foundation.NSBundle.MainBundle.BundleIdentifier}/callback",
    Browser = new PlatformWebView(options.Controller),
#elif __ANDROID__
    RedirectUri = options.RedirectUri ?? $"{packageName}://{options.Domain}/android/{packageName}/callback".ToLower(),
    Browser = new PlatformWebView(options.Activity),
```

**Comment by jerriep** [comment]
This is fixed in the upcoming V2:
https://github.com/auth0/auth0-oidc-client-net/blob/v2/src/Auth0.OidcClient.Shared/Auth0Client.cs#L40

At the time I think I did it because there were strict requirements on the constructor parameters being passed in (i.e. the `Activity` and `UIViewController` instances).

In any case, I am awaiting feedback from testers on V2, so, hopefully, it is not too far off anymore.

**Comment by badavid85** [comment]
Ok, thanks for your response.
Do you have an estimated date on v2 release? Or is there any way to use the prerelease package in the meantime?

**Comment by jerriep** [comment]
Nothing definite. It depends on the feedback I get. Hopefully, we can do it this month.

**Comment by jerriep** [comment]
@badavid85 This was fixed in v2.0.0
